src/OrcaSlicer.cpp

```diff
@@ -1051,6 +1051,10 @@ int CLI::run(int argc, char **argv)
     // Also on Linux, we need to tell Xlib that we will be using threads,
     // lest we crash when we fire up GStreamer.
     XInitThreads();
+
+    // https://github.com/prusa3d/PrusaSlicer/issues/12969
+    ::setenv("WEBKIT_DISABLE_COMPOSITING_MODE", "1", /* replace */ false);
```

Comment:
Orca fixed this issue with it before, but was removed due to crashing issues on Linux.

https://github.com/SoftFever/OrcaSlicer/pull/5202#discussion_r1584666530

Reply:
I see. Then let's wait and see how PrusaSlicer works.